arrow-core-data/src/main/kotlin/arrow/core/SortedMapK.kt

```diff
@@ -1,12 +1,18 @@
 package arrow.core
 
 import arrow.Kind
-import arrow.higherkind
 import arrow.typeclasses.Applicative
 import arrow.typeclasses.Show
 import kotlin.collections.flatMap
 
-@higherkind
+class ForSortedMapK private constructor() { companion object }
```

Comment:
All these should be deprecated without replacement now that they are spelled out in the sources.

Reply:
yes!! there are already ongoing PRs for Const, Eval, Ior deprecating them.. so they are being deprecated for each typeclass separately.. but in order to remove kapt and unblock one issue I have, this PR removing kapt is needed. Deprecation of all those will come soon too!